Tests/UnitTests/Scripts/tests.js

```diff
@@ -1,5 +1,5 @@
 ﻿// Set this to true to make attaching a debugging easier.
-const waitForDebugger = true;
+const waitForDebugger = false;
```

Comment:
Why do we need to set this flag to false? Will this impact debugging on other platforms?

Reply:
Setting this to true will wait for the debugger. We don't want that for normal, non-debugging cases (e.g., running in the CI).